Dockerfile

```diff
@@ -5,9 +5,10 @@ USER 0
 
 # Install dependencies and clean cache to make the image cleaner
 # also remove unused packages added by ubi8/s2i-base.
-RUN yum install -y hostname shared-mime-info jq && \
-    yum remove -y mariadb-connector-c-devel npm openssh && \
-    yum clean all -y && \
+RUN dnf update -y && \
```

Comment:
@vkrizan @LightOfHeaven1994 not sure if this is a good practice :thinking:

Reply:
Well, yes and no. It increases image size. I'd say that updates should happen on base image. However, I've seen this on some images (not sure if they were production ones). My conclusion would be to trust base image.